### ISSUE TITLE
Detections from train and val

### ISSUE THREAD

**Issue by bstandaert** [hsiangwei0903/Deep-EIoU#11]
Hi,
Thank you for your great work, this is very interesting. I would be interested in getting the detections as you have provided for the test but also for the train and validation set. Would you mind either sharing these (in .npy format) or either explaining how you obtained them for the test set.
Thank you very much.

**Comment by hsiangwei0903** [comment]
We have released the detection for the test set. For the training and validation set the ground truth detection is provided in the SportsMOT dataset. You can modify demo.py to get the detection (in any format, according to how you modify the code).
